File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    짝수합 = 0
    홀수합 = 0
    
    for i in range (0 , len(num_list)) :
        if i % 2 == 0 :
            홀수합 += num_list[i]
        else : 짝수합 += num_list[i]
    
    return 홀수합 if 홀수합 > 짝수합 else 짝수합

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int H = 0;
        int J = 0;
        
        for (int i = 0; i < num_list.Length; i++) {
            if (i % 2 == 1)
                H += num_list[i];
            else
                J += num_list[i];
        }
        
        return (H > J) ? H : J;
    }
}